packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.tsx

```diff
@@ -107,18 +109,18 @@ export class Checkbox extends BaseComponent<ICheckboxProps, ICheckboxState> impl
         id={ this._id }
         role='checkbox'
         type='button'
-        className={ classNames.root }
+        className={ this._classNames.root }
         onClick={ this._onClick }
         onFocus={ this._onFocus }
         onBlur={ this._onBlur }
         aria-checked={ isChecked }
         aria-disabled={ disabled }
       >
-        <label className={ classNames.label } htmlFor={ this._id } >
-          <div className={ classNames.checkbox }>
-            <Icon iconName='CheckMark' className={ classNames.checkmark } />
+        <label className={ this._classNames.label } htmlFor={ this._id } >
+          <div className={ this._classNames.checkbox }>
+            <Icon iconName='CheckMark' className={ this._classNames.checkmark } />
           </div>
-          { label && <span className={ classNames.text }>{ label }</span> }
+          { label && onRenderLabel(this.props, this._onRenderLabel) }
```

Comment:
![image](https://user-images.githubusercontent.com/9662034/29040203-fd3666ee-7b62-11e7-947a-c924e9872ff7.png)



  private _rnderPersonaNoLabel() {
    const DEFAULT_IMAGE_URL = '/_layouts/15/userphoto.aspx?size=S&accountname=';
    return <Persona imageUrl={ DEFAULT_IMAGE_URL } size={ PersonaSize.extraSmall } imageInitials="TE" />;
  }

        <Checkbox
          label='Persona no label checkbox'
          styles={ styles }
          onRenderLabel={ this._rnderPersonaNoLabel }
        />